dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.restConnector.javaee-2.0.feature

```diff
@@ -0,0 +1,15 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+
+symbolicName = io.openliberty.restConnector.javaee-2.0
```

Comment:
This name looks odd to me because there is no such feature as `javaee-2.0`. I suggest calling this `io.openliberty.restConnector2.0-servlet`.

Reply:
But server3.0 is odd because it is enabled for all servlet versions in javaee (3.0, 3.1, 4.0).  If this is to change I suggest changing it to `io.openliberty.restConnector.javaee`

Reply:
OK, that is a little less confusing.

Reply:
Changed filename to `io.openliberty.restConnector2.0.javaee`. Thanks!